pkg/secrets/aws.go

```diff
@@ -0,0 +1,72 @@
+package secrets
+
+import (
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	// AWSSecretType represent the secret type for AWS credentials.
+	AWSSecretType string = "secrets.kanister.io/aws"
+
+	// AWSAccessKeyID is the key for AWS access key ID.
+	AWSAccessKeyID string = "awsAccessKeyID"
```

Comment:
Why not use the same values that the AWS SDK and CLI use? That's what mot people would expect these (environment) variables to be named, and thus to be the corresponding keys in the secret

Reply:
@tdmanv suggested we are using camel case with secret data fields in Kanister.

Reply:
I think I said lowercase. The field names I've seen elsewhere in our code are:
```
    access_key_id: XXX
    secret_access_key: XXX
```

Reply:
@tdmanv lower case is _"surprising"_, what's the rationale for it?
With the given key names, it would not be possible to _directly_ [export all key-value pairs in a secret as environment variables](https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#configure-all-key-value-pairs-in-a-secret-as-container-environment-variables).